### PR TITLE
Fix check harness

### PR DESCRIPTION
**Pull request by mezis**
* Actually run the check harness
* Run environment-independent checks once

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -39,3 +39,8 @@ Style/DotPosition:
   Enabled: false
 Metrics/AbcSize:
   Max: 30
+
+Rubocop/Metrics/MethodLength:
+  Max: 20
```
Comment (danielcooper) [inline .rubocop.yml]: Why the bump?

Comment (mezis) [inline .rubocop.yml]: Because the default (10 lines) is fairly ridiculous (in my humble opinion!)

**Comment by mezis** [comment]
@humzashah given you PR #50 went in before this, I've had to touchup the tests a bit. Mind having a look?

File: .rubocop.yml (hunk 1 above)
Comment (humzashah) [inline .rubocop.yml]: agreed with @mezis

File: lib/roo_on_rails/checks/base.rb [hunk 2]
```diff
@@ -20,10 +20,13 @@ def initialize(options = {})
         @fix     = @options.delete(:fix) { false }
         @context = @options.delete(:context) { Hashie::Mash.new }
         @shell   = @options.delete(:shell) { Shell.new }
+        @dry_run = options.fetch(:dry_run, false)
       end
 
       def run
+        # $stderr.puts "#{signature.join '/'}#run (dry_run = #{@dry_run})"
```
Comment (humzashah) [inline lib/roo_on_rails/checks/base.rb]: sorry if i'm missing something obvious: what's this comment for?

Comment (mezis) [inline lib/roo_on_rails/checks/base.rb]: it's a leftover! sorry ^_^

**Pull request merged**